### PR TITLE
Improves snap metadata content

### PR DESCRIPTION
**Pull request by gruyaume**
- Improves snap metadata content
- Simplifies the README.md

### CODE REVIEW

**Review by ghislainbourgeois — APPROVED** [review]

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -2,9 +2,24 @@ name: srsran
...
+  - srsUE: a full-stack 4G and 5G NSA / SA UE application
+  - srsENB: a full-stack 4G eNodeB with 5G NSA / SA gNodeB capabilities
+  - srsEPC: a light-weight 4G EPC implementation with MME, HSS and S/P-GW
+
+  All srsRAN software runs in linux with off-the-shelf compute and radio hardware.
 grade: devel
-confinement: devmode
+confinement: classic
+icon: snap/srsran.png
+website: https://www.srslte.com/
```
Comment (sanchezfdezjavier) [inline snap/snapcraft.yaml:19]: `For more information, see www.srsran.com` and `website: https://www.srslte.com/`
We are pointing to different URLs, they point to the same website anyways, I guess it's not a problem.
Feel free to resolve the comment it's super nitpick.

Comment (gruyaume) [reply snap/snapcraft.yaml]: Good catch, fixed

**Review by sanchezfdezjavier — APPROVED** [review]

File: snap/snapcraft.yaml [hunk 2]
```diff
@@ -2,9 +2,24 @@ name: srsran
...
+
+  The srsRAN suite currently includes:
+  - srsUE: a full-stack 4G and 5G NSA / SA UE application
+  - srsENB: a full-stack 4G eNodeB with 5G NSA / SA gNodeB capabilities
+  - srsEPC: a light-weight 4G EPC implementation with MME, HSS and S/P-GW
+
+  All srsRAN software runs in linux with off-the-shelf compute and radio hardware.
 grade: devel
-confinement: devmode
+confinement: classic
```
Comment (saltiyazan) [inline snap/snapcraft.yaml:17]: based on the [documentation](https://snapcraft.io/docs/snap-confinement), it's not clear to me that devmode and classic provide the same level of access.
I'm testing it and I see that the behaviour is not the same.
Please don't merge yet.

Comment (gruyaume) [reply snap/snapcraft.yaml]: Reverted to using devmode

**Review by saltiyazan — APPROVED** [review]

**Pull request merged**